### PR TITLE
drafts and versions

### PR DESCRIPTION
**Pull request by ppanero**
- closes https://github.com/zenodo/zenodo-rdm/issues/241

### CODE REVIEW

File: Pipfile.lock
Comment (alejandromumo) [inline Pipfile.lock:1]: This was updated on master and has conflicts. 
is this the version I've sent you?

File: site/zenodo_rdm/migrator/streams.yaml [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-  load:
-    db_uri: postgresql://zenodo:zenodo@localhost:5432/zenodo
-    tmp_dir: /path/to/tmp
-communities:
-  extract:
-    filepath: /path/to/communities.jsonl
-  load:
-    db_uri: postgresql://zenodo:zenodo@localhost:5432/zenodo
-    tmp_dir: /path/to/tmp
+# users:
```
Comment (zzacharo) [inline site/zenodo_rdm/migrator/streams.yaml:1]: leftover, you need to revert it back to its initial state

File: site/zenodo_rdm/migrator/extract.py [hunk 2]
```diff
@@ -100,6 +100,25 @@ def run(self):
     ) TO STDOUT;
 """
 
+EXTRACT_DRAFTS_SQL = """
```
Comment (ppanero) [inline site/zenodo_rdm/migrator/extract.py:103]: this will be updated by alex

File: site/zenodo_rdm/migrator/transform/records.py [hunk 3]
```diff
@@ -10,17 +10,17 @@
...
 
 class ZenodoRecordTransform(RDMRecordTransform):
     """Zenodo to RDM Record class for data transformation."""
 
     def _communities(self, entry):
-        communities = entry["json"].get("communities")
-        if communities:
-            slugs = [slug for slug in communities]
-            return {"ids": slugs, "default": slugs[0]}
+        # communities = entry["json"].get("communities")
```
Comment (zzacharo) [inline site/zenodo_rdm/migrator/transform/records.py:20]: can we put this back or there is something wrong with drafts that belong already to communities?

File: site/zenodo_rdm/migrator/extract.py (hunk 2 above)
Comment (slint) [reply site/zenodo_rdm/migrator/extract.py:103]: No, the one here is correct, it's the same I have :)

**Pull request merged**